components/ui/carousel.tsx

```diff
@@ -241,7 +241,7 @@ const CarouselPrevious = React.forwardRef<
       onClick={scrollPrev}
       {...props}
     >
-      <FaChevronLeft className="text-neutral-100" size={40} />
+      <FaChevronLeft className={cn('text-neutral-100', className)} size={40} />
```

Comment:
`className` is already put in the wrapping `<Button>` tag. It won't be put in both places.

Reply:
yes sir I had figured it out just after I msged you in discord, but if I had not done this then I would have to pass the color from slideshow which is not related to my branch and as you once notified not to change anything unrelated to my branch I let it be there

Reply:
removing it will work but then slideshow have to pass the  color for the arrow which I thing is not being done currently